fdt.hh

```diff
@@ -417,8 +418,18 @@ class node
 	 * name followed by an at symbol.
 	 */
 	std::string unit_address;
-	struct node_flag_bits {
+	struct node_flag_bits
```

Comment:
Is there a reason for this to be a separate structure, rather than just having the bitfield in the enclosing scope?

Reply:
I think I might have had a good reason at some point (back when I first started on this, months ago), but it's pretty clear now that it's wrong.